### DISCUSSION TITLE
Support as well the old booting method

### DISCUSSION THREAD

**Discussion by dashohoxha** [ltsp/ltsp#180] [Q&A]
I am using VirtualBox for testing LTSP (https://gitlab.com/Virtual-LTSP/VirtualBox), but it seems that it does not work with iPXE. This is the code that I am using for the client: https://gitlab.com/Virtual-LTSP/VirtualBox/blob/bionic/scripts/ltsp-client.sh#L62

It would be great if the new LTSP supports (out of the box) the old booting method as well. Maybe there are other old computers that suffer from this issue too (i.e. cannot boot with iPXE).

**Comment by alkisg** [comment]
Hi Dashamir,

I've transferred the issue to the [community issues](https://github.com/ltsp/community/issues) as it doesn't involve changes to the LTSP code base.

The new LTSP does support VirtualBox out of the box; in fact while developing, I'm first testing with VirtualBox, then with real clients, then in schools.

Moreover, VirtualBox itself embeds and uses a minimal version of iPXE.

If you're having problems booting a virtual client, try first with https://boot.ipxe.org/ipxe.iso, then check your VM settings, that it's using bridged networking mode; if those don't solve the issue, paste a screenshot of the boot process so that we see where it stucks.

**Comment by dashohoxha** [comment]
I will have a look at it as soon as I find some time.
